app/schemats/posts.py

```diff
@@ -0,0 +1,66 @@
+from datetime import datetime
+from decimal import Decimal
+from typing import List, Optional
+
+from app.models.account_type import AccountType
+from app.models.post import PostType
+from fastapi import Form, Query
+from pydantic import BaseModel
+
+# from .users import User
```

Comment:
ok funny comment

Reply:
This is a problem because we have there recursive importing. Idk how to fix it. :-(